source/content/guides/decoupled-sites/04-site.md

```diff
@@ -16,13 +16,15 @@ reviewed: "2022-09-19"
 
 ## Before You Begin
 
-You should have the **Decoupled** tab enabled in your dashboard after onboarding. To begin creating a Front-End Site, ensure you meet the following prerequisites:
+You should have the **Decoupled** tab enabled in your Sites dashboard after onboarding. To begin creating a Front-End Site, ensure you meet the following prerequisites:
 
-* Your site repository is in GitHub.
+* You have a GitHub account
```

Comment:
Probably worth mentioning as well that if they are bringing an existing codebase that the site repository exists on GitHub.

Reply:
To me that's a specific qualification for doing the Import workflow, and that section has its own pre-reqs. I think that's a bit of a self-fulfilling one, as if they are importing a repo they must have a repo. Happy to change it if you see it differently of course